### ISSUE TITLE
WACCM-MA CARMA 2deg F-case model run from 1980-2014

### ISSUE THREAD

**Issue by tilmes** [NCAR/CARMA_dev#1]
The purpose of this run is to evaluate the new WACCM-MA 2deg model in a free running model case with prescribed SST and compare to the earlier simulations with WACCM-MA 2deg model and to observations.  The model will start from the 2deg WACCM MAM version, and will start as an aerosol spin-up run (no CARMA aerosols), since we don't have any initial conditions for this period.  The code base will be based on a new tag from /glade/work/fvitt/camdev/carma_trop_strat , that will include some updates that will be added  soon (in further discussions).
Model costs for this run are approx. 11K / year = 605K total.

**Comment by tilmes** [comment]
I am proposing to use a nudged QBO for these runs, this is expected to showing better results. We need to keep in mind that these model runs with SAI will not change the QBO.

**Comment by tilmes** [comment]
@dan-visioni @drmikemills  Who  ran Nick's 2deg model run? Could you add the case name and the location of the restart files, so we can setup a similar case with CARMA and can compare to the WACCM 2deg MAM4 run?

**Comment by tilmes** [comment]
Case Setup: 2deg FWmaCARMAHIST compset
Code Base: /glade/work/fvitt/camdev/carma_trop_strat09
CaseDir: /glade/p/cesm/chwg_dev/tilmes/cases/carma/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat09.1980_2014
- nudged QBO
- Zhau Nucleation scheme
- 7TgS Mt Pinatubo regional injection

**Comment by tilmes** [comment]
For comparisons the similar MAM4 runs with fixed SSTs are under
/glade/work/cmip6/cases/DECK_WACCM_2deg

f.e21.FWmaHIST_BGC.f19_f19_mg17.CMIP6-AMIP-WACCM-MA-2deg.001
f.e21.FWmaHIST_BGC.f19_f19_mg17.CMIP6-AMIP-WACCM-MA-2deg.002
f.e21.FWmaHIST_BGC.f19_f19_mg17.CMIP6-AMIP-WACCM-MA-2deg.003

Output is here: /glade/campaign/collections/cmip/CMIP6/timeseries-cmip6

**Comment by tilmes** [comment]
@zywshoon @dan-visioni 
The first 12 years (1975 - 1987) have been  completed and at least SAOD (blue) looks good compared to GloSSAC:
<img width="726" alt="image" src="https://user-images.githubusercontent.com/22222754/204345929-6677d60b-6648-4443-b8ef-4fead7b0abfb.png">
I am continuing to run till 1990 and then do some initial diagnostics to see if this looks similar to the WACCM MA 2x model in Nick's paper!

**Comment by tilmes** [comment]
here are some initial diagnostics:
https://acomstaff.acom.ucar.edu/tilmes/amwg/carma/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat09.1980_2014-obs.1979_1987/

**Comment by tilmes** [comment]
I am setting up a new run that is the same to this run but uses MAM4 for comparisions:
CaseDir: /glade/p/cesm/chwg_dev/tilmes/cases/carma/FWmaMAM4HIST.f19_f19_mg17.carma_trop_strat09.1980_2014

The differences to the CMIP6 simulations in Nick's paper and the new MAM4 run (that is similar to CARMA):
- the CMIP6 simulations don't include the new convective removal scheme by Shan et al, 2021, so we will have more aerosols in the  upper stratosphere (like BC and sea-salt) in CMIP6 runs than the new runs. 
- Different DMS emissions, for the new runs we use oceanic DMS emissions from an machine learning approach
- CFC11STAR is not used in the new runs.

**Comment by tilmes** [comment]
Here are the diagnostics for CARMA vs MAM4 (1979-1987) (will run longer diagnostics when more data are available)
(https://acomstaff.acom.ucar.edu/tilmes/amwg/carma/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat09.1980_2014-FWmaMAM4HIST.f19_f19_mg17.carma_trop_strat09.1980_2014/)

I see a large difference in the High Cloud forcing and changes in the SW cloud forcing. I also see changes to the tropopause temperature and smaller changes in ozone. Changes in SW cloud forcing are mostly due to reduced tropospheric aerosols like sea-salt and dust  in the model.

Differences in the high clouds are likely due to the difference in the mixed-cloud scheme. I am starting to look into this in more detail next, in particular, adding some bug fixes to MAM4 and eventually also updating the CARMA. This will be important before we do the B-case tuning.

**Comment by tilmes** [comment]
Here are new diagnostics closer to present day, we need to look into more details.. why is water vapor different, why is ozone  in the stratosphere different?
https://acomstaff.acom.ucar.edu/tilmes/amwg/carma/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat09.1980_2014.1995_2004-FWmaMAM4HIST.f19_f19_mg17.carma_trop_strat09.1980_2014.1995_2004/

**Comment by tilmes** [comment]
CARMA (blue) MAM4 (red), free running model (fixed SSTs) looks very good. MAM4 overestimates AOD with 7TgS injections: 
<img width="727" alt="image" src="https://user-images.githubusercontent.com/22222754/207488725-fc125717-3cfe-43ff-b04c-6cf2a4374dfe.png">

**Comment by tilmes** [comment]
We discovered today, that the CARMA implementation has a bug in the radiation scheme (thanks to Francis), in particular the long-wave absorption. I plotted the temperature in the stratosphere 24km, tropics, zonal average, between CARMA (black) and MAM4 (red) and we are indeed show cooler temperatures in CARMA over the Mt Pinatubo period, with differences up to 5K.
<img width="962" alt="image" src="https://user-images.githubusercontent.com/22222754/218864033-0d5a6af8-9ea0-43f0-b404-4b54f6aadf6e.png">
and Differences:
<img width="908" alt="image" src="https://user-images.githubusercontent.com/22222754/218864098-75cba98c-12ed-4c4a-a1e2-dbd8a35741e1.png">

I also looked at long-wave heating: QRL_TOT
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/22222754/218864796-3da70c6e-3e2e-4f88-afd0-f88954735113.png">
QRS
<img width="1026" alt="image" src="https://user-images.githubusercontent.com/22222754/218865289-607585db-afde-47c8-94e1-d2f3e3bd3cc6.png">

We will fix this, it is good, we have not started our next long CARMA run yet :)

**Comment by dan-visioni** [comment]
I believe you if you say there's a bug, but the difference in temperature could just be driven by the fact MAM4 sees larger particles in Marc 1992 (your Fig. 7)?
Also, not clear to me at his moment why you don't see a warming at 25km after Pinatubo, but only in 1993? Am I missing something?

**Comment by tilmes** [comment]
Yes, it is a bug; there was a wrong multiplication in the code. The temperature is significantly warmer in MAM between mid-1991 and 1993 (the second plot is a difference plot). Sorry maybe may QRL and QRS plots are messed up, I will check and if so, I will replace them. Thanks!

**Comment by tilmes** [comment]
@dan-visioni @fvitt @cbardeen 
Daniele had a good point above, I plotted the temperature change between the CARMA and MAM4 run that I did over Mt Pinatubo and we see less heating in CARMA. I thought it is because of the absorption,  however, Dan pointed out that the mass in both are very different, which  is true. So, I plotted now the two  nudged runs that had similar burden (CARMA black, MAM4 red) over the tropics in March 1992, and QRL is actually very similar. 
<img width="656" alt="image" src="https://user-images.githubusercontent.com/22222754/219161497-8efc9ce7-1f01-4083-9335-166ba2c99c14.png">
and QRS
<img width="647" alt="image" src="https://user-images.githubusercontent.com/22222754/219162345-805c25ad-7669-4e8a-8ac5-ea0e2e6a638b.png">
Unfortunately, I don't have absorption in the run, so I can't look at this.
This is temperature, but it's nudged. 
<img width="657" alt="image" src="https://user-images.githubusercontent.com/22222754/219165055-1399cd88-d4b6-4de5-9549-5eb4d748a113.png">

Therefore, I am not sure any more, if we are having a problem in absorption, since the QRL is very similar. 
We could test it and also go back to the discussion from yesterday and see if someone has an idea?

**Comment by tilmes** [comment]
I redid a run with MAM4 adding absorption, here it is for for March 1992:
CARMA (left), MAM4 (right)
<img width="1486" alt="image" src="https://user-images.githubusercontent.com/22222754/219389681-eab5a540-948f-4384-8495-d6597529fc3e.png">
and AODABS
<img width="1477" alt="image" src="https://user-images.githubusercontent.com/22222754/219390081-768c5e79-495c-430e-8876-8a770a740334.png">

This is the absorption profile for CARMA (left) and MAM (right)
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/22222754/219391009-f8e1dd58-af5b-455b-b0e0-4a0f6a69acf0.png">
and the difference (MAM is red)
<img width="672" alt="image" src="https://user-images.githubusercontent.com/22222754/219391146-2ec3d798-ab0c-4cf7-b59b-202a89263075.png">
